src/components/ResourceLink.tsx

```diff
@@ -11,7 +11,7 @@ interface Props {
 const ResourceLink: FC<Props> = ({ type, value, to }) => {
   return (
     <Link
-      className="p-chip is-inline is-dense resource-link"
+      className={"p-chip is-inline is-dense resource-link"}
```

Comment:
Why? :grin:

Reply:
was making some changes in here but reverted, seems like I missed some haha